### PR TITLE
Feature/meta modal updates

### PR DESCRIPTION
**Pull request by edbrett**
## Overview

A housekeeping PR to bring the metadata modals up to speed.
- Add the exceptions for the country pages meta keys to the widget header
-- treeCover widget uses natural forest key when it has plantations
-- primary forests uses the iso code to correctly find its meta key
- Update the main map modal to use a consistent design like the country pages (full page scrolling modal with updated table styles

### CODE REVIEW

File: app/assets/stylesheets/modules/_modal_new.scss [hunk 1]
```diff
@@ -49,170 +41,107 @@ $height: auto;
...
+  border-radius: 0px;
+  outline: none;
+  padding: 30px;
+  margin: auto;
+  max-width: 800px;
+
+  @media (min-width: $br-mobile){
+    padding: 50px;
+    padding-top: 40px;
+    padding-bottom: 30px;
```
Comment (pjosh) [inline app/assets/stylesheets/modules/_modal_new.scss]: unnecessary overwriting. Better with: `padding: 40px 50px 30px 50px;`

File: app/assets/stylesheets/modules/_modal_new.scss [hunk 2]
```diff
@@ -447,22 +381,30 @@ $height: auto;
...
           background: none;
+          padding-left: 50px;
         }
       }
 
       dd {
         padding: 10px;
         font-size: 13px;
+        padding-right: 30px;
+        padding-left: 30px;
```
Comment (pjosh) [inline app/assets/stylesheets/modules/_modal_new.scss]: unnecessary overwriting. Better with: `padding: 10px 30px;`

File: app/assets/stylesheets/modules/_modal_new.scss [hunk 3]
```diff
@@ -447,22 +381,30 @@ $height: auto;
...
+
         @media (min-width: $br-mobile){
           display: table-cell;
           padding: 10px 10px 10px 15px;
           border-left: 1px solid $border;
           border-bottom: 1px solid $border;
+          padding-right: 50px;
+          padding-left: 10px;
```
Comment (pjosh) [inline app/assets/stylesheets/modules/_modal_new.scss]: unnecessary overwriting. Better with: `padding: 10px 50px 10px 10px;`

File: app/assets/stylesheets/modules/_modal_new.scss [hunk 4]
```diff
@@ -769,8 +711,7 @@ $height: auto;
     .modal-table {
       border-collapse: collapse;
       font-size: 11px;
-      margin: 20px 0;
-      width: 100%;
+      // margin: 20px 0;
```
Comment (pjosh) [inline app/assets/stylesheets/modules/_modal_new.scss]: commented code

**Comment by edbrett** [comment]
Thanks for the feedback! Fixed 💨 . The file is a bit of a mess but it is a lot of work to tidy :(.

**Pull request merged**